.circleci/config.yml

```diff
@@ -57,39 +58,16 @@ jobs:
       - run:
           name: Get test env vars
           command: make env
+      # run verify and then save cache.
+      # this ensures that the Cypress verified status is cached too
+      - run: make cypress-verify
 
       - *cache_node_modules
 
       - persist_to_workspace:
           root: *workspace_root
           paths:
             - .
-  cypress-build:
```

Comment:
Highlight #1 - removed a build step which was more or less just running npm install again + a cli tool to check the cypress binary got set up correctly (which I think is a piostinstall script in npm install anyway). Have brought that into the main install job and tweaked the statements of what gets put in the cache to makes sure the cypress binary is included